### PR TITLE
Updated strings from Crowdin 20221109

### PR DESCRIPTION
**Pull request by mikehardy**
Should be translations-only, no key add/remove, needs review

### CODE REVIEW

File: AnkiDroid/src/main/res/values-de/03-dialogs.xml [hunk 1]
```diff
@@ -258,13 +258,8 @@
...
-    <string name="scoped_storage_require_user_to_accept_risk">        Due to Android privacy changes, your current storage location will be inaccessible if the app is uninstalled.
-        \n\nWe need to migrate your data to a location private to the AnkiDroid app.
-        \n\nData in this private location may be deleted if the app is uninstalled.
-        \n\nYour current storage location is %1$s
-        \n\nThe private storage location will be %2$s
-        \n\nPlease confirm that you will regularly make backups of this folder.
-    </string>
-    <string name="manual_backup">Manual backup</string>
-    <string name="remind_me_later">Remind me later</string>
+    <string name="scoped_storage_require_user_to_accept_risk">Aufgrund von Änderungen in der Privatsphäre von Android wird dein aktueller Speicherort nicht mehr aufrufbar sein, solltest du die App deinstallieren.
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-de/03-dialogs.xml:261]: This one is incorrect. Misses interpolation sequences

File: AnkiDroid/src/main/res/values-ur/02-strings.xml [hunk 2]
```diff
@@ -157,25 +157,26 @@
...
+    <string name="import_log_failed_validate">apkg کی توثیق ناکام ہوگئی</string>
     <string name="import_log_insufficient_space_error">There is not enough space to import the package.\nNeeded - %1$s \nAvailable - %2$s.\n\nHaving Trouble? Try to free space on your phone if possible. Alternatively, sync uses less space than import. You may try importing the package with a device with more space (for example Anki Desktop) and then sync it to this device.</string>
     <string name="import_log_file_cache_cleared">Error importing file, likely a cache clear during processing.\nPlease try again</string>
     <string name="import_succeeded_but_check_database">Data import succeeded but post-import cleanup failed. Run check database later. Root cause: %s</string>
     <string name="import_error_corrupt_zip">The apkg file is corrupt. Please delete and re-download it.\n\nRoot cause: %s</string>
     <string name="import_error_handle_exception">Failed to import file\n\n%s</string>
-    <string name="import_error_not_apkg_extension">Filename “%s” doesn’t have .apkg or .colpkg extension</string>
+    <string name="import_error_not_apkg_extension">فلٹر شدہ ڈیک</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-ur/02-strings.xml:179]: This is incorrect, missing format string sequence

Comment (mikehardy) [reply AnkiDroid/src/main/res/values-ur/02-strings.xml]: @oyeraghib this was your translation - I reverted it FYI - needs a second attempt on crowdin

File: AnkiDroid/src/main/res/values-ur/02-strings.xml [hunk 3]
```diff
@@ -157,19 +157,20 @@
...
+    <string name="info_rate">AnkiDroid کی درجہ بندی کریں</string>
+    <string name="import_title">درآمد</string>
+    <string name="import_message_add" comment="Adding a new deck to the collection (import)">شامل کریں</string>
+    <string name="import_message_replace_confirm">یہ آپ کے موجودہ مجموعہ کو حذف کر دے گا اور اسے فائل کے ڈیٹا سے بدل دے گا %s</string>
     <plurals name="import_files_message_add_confirm">
-        <item quantity="one">Add “%s” to collection? This may take a long time</item>
-        <item quantity="other">Do you want to import the following files:\n%s\nThis may take a long time</item>
+        <item quantity="one">مجموعہ میں “%s” شامل کریں۔ یہ ایک طویل وقت لگ سکتا ہے</item>
+        <item quantity="other">Do you want to import the following files:\n%s\nThis may take a long time
+کیا آپ درج ذیل فائلوں کو درآمد کرنا چاہتے ہیں:\n%s\nیہ ایک طویل وقت لگ سکتا ہے</item>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-ur/02-strings.xml:167]: @oyeraghib these also caused string format errors, and I reverted them :thinking:

Comment (oyeraghib) [reply AnkiDroid/src/main/res/values-ur/02-strings.xml]: I see, seems like I just reverted the parts. The last part should come before. No worries I will take a look at crowdin and fix it.

**Pull request merged**